core/src/main/java/org/apache/iceberg/avro/Avro.java

```diff
@@ -124,6 +125,11 @@ public WriteBuilder meta(Map<String, String> properties) {
       return this;
     }
 
+    public WriteBuilder overwrite(boolean newOverwrite) {
```

Comment:
I suggest also adding `overwrite()` that sets `this.overwrite = true`. That way users can avoid passing a boolean constant.

Reply:
Added.